src/ImageSequencer.js

```diff
@@ -245,6 +245,32 @@ ImageSequencer = function ImageSequencer(options) {
     return `sequencer -i [PATH] -s ${cliStringSteps} -d '${JSON.stringify(cliOptions)}'`
   }
 
+  // Checks if input is a string of comma separated module names
+  function detectSyntax(str) {
+    let result = str.includes(',') ? true : false
+    return result
+  }
+
+  // Parses input string and returns array of module names
+  function parseSyntax(str) {
+    let stringifiedNames = str.replace(/\s/g, '')
+    let moduleNames = stringifiedNames.split(',')
+    return moduleNames
+  }
+
+  // imports string of comma separated module names to sequencer steps
+  function stringToStep(str) {
```

Comment:
Just wondering if this is slightly redundant with `stringToJSON()`?

https://github.com/publiclab/image-sequencer/blob/ac57b32ec636b7b38bae5d9b53a5d0dd6c3feadd/src/ImageSequencer.js#L289-L297

Reply:
I took the `','` check out of the `stringToJSON()` and generalised it in `detectSyntax` and `parseSyntax` for this reason. I think for expanding this functionality, wherever we need to detect and parse a comma-separated string, we could use the new functions. For example, here `stringToJSON()` checks for comma separated string and then returns a JSON step. The if condition would have to be checked a lot, in say, support for arrays and object-based steps. I'll push more code where I think this will be a modular approach. Incoming!

Reply:
OK, shall we just add in the docs the difference between them? Also, should it be `stringToSteps`?